server/lib/adapters/mysql.js

```diff
@@ -74,13 +75,13 @@ export function runQuery(client, query, options = {}) {
       }
 
       // Run the query.
-      return client.query(params, (queryError, rows) => {
+      return client.query(params, (queryError, rows, fieldPackets) => {
         client.end();
```

Comment:
@michaloo Yes, actually we were calling twice client.end() not only for mysql connector but for postgres too. The difference is that postgres does not throw an error if we call it twice - mysql does. This fixed it in this PR.